packages/gatsby-theme-newrelic/src/components/SEO.js

```diff
@@ -38,6 +38,14 @@ const SEO = ({ title, location, children }) => {
       ? location.pathname
       : location.pathname.replace(new RegExp(`^\\/${locale.locale}`), '/');
 
+  const getSwiftypeSiteType = () => {
+    const nrSubDomain = /.*\.newrelic\.com/.test(location.hostname)
```

Comment:
I worry that `hostname` is not available on a build since it doesn't execute in a browser. `pathname` makes sense since Gatsby knows which page it is currently building. Can we verify this works as expected when the site is built and not just at runtime?

Reply:
i ran the the build for the demo workspace and didn't see any errors. think that's sufficient?

Reply:
In this case, I'm not worried about errors, I'm worried that the value will be either `null` or `undefined` (depending on how that location comes back in the SSR case). Since `location` is an object, accessing the `hostname` property is always going to succeed.

Reply:
ah yes. thanks for catching that, @jerelmiller ! `location.hostname` is indeed `undefined` during SSR. i know we've done similar things to handle something like this during SSR. i'll look for that. in the meantime, let me know how you've handled that in the past / know where i should look

Reply:
i suppose i could use the `siteUrl` from the static query in the component. unless there's something more reliable

Reply:
i went that route of using `siteUrl` since that should be fairly reliable AND i overlooked it being used for other `<link>` elements in the component